chat-plugins/info.js

```diff
@@ -109,6 +109,19 @@ var commands = exports.commands = {
 		if ((user === targetUser || user.can('makeroom')) && privaterooms) {
 			buf += '<br />Private rooms: ' + privaterooms;
 		}
+
+		if (user.can('lock') || (user.can('roomban') && Object.keys(room.users).indexOf(targetUser.userid) >= 0)) {
```

Comment:
It would be faster to do <code>(user.can('roomban') && targetUser.userid in room.users)</code>.